src/components/Dashboard/RewardsModule.js

```diff
@@ -422,4 +394,35 @@ const FeeSection = ({ children }) => {
   )
 }
 
+function getTotalFees(fees, key = 'amount') {
+  return Array.isArray(fees)
+    ? fees.reduce((acc, fee) => acc.add(fee[key]), bigNum(0))
+    : bigNum(0)
+}
+
+function getTotalSettledFees(arbitrableFees, appealFees) {
+  return [arbitrableFees, appealFees]
+    .map(fees => getTotalFees(fees, 'settledAmount'))
+    .reduce((acc, amount) => acc.add(amount), bigNum(0))
+}
+
+const useTotalFeeRewards = (arbitrableFees, appealFees, subscriptionFees) => {
```

Comment:
We should probably default each variable to `[]`; it looks like from above that we're using `?.` and a null value could be given.